pkgs/development/libraries/boehm-gc/default.nix

```diff
@@ -30,6 +30,21 @@ stdenv.mkDerivation (finalAttrs: {
   ++ lib.optional enableMmap "--enable-mmap"
   ++ lib.optional enableLargeConfig "--enable-large-config";
 
+  # This stanza can be dropped when a release fixes this issue:
+  #   https://github.com/ivmai/bdwgc/issues/376
+  # The version is checked with == instead of versionAtLeast so we
+  # don't forget to disable the fix (and if the next release does
+  # not fix the problem the test failure will be a reminder to
+  # extend the set of versions requiring the workaround).
+  makeFlags = if (stdenv.hostPlatform.isPower64 &&
```

Comment:
In cases of "else null", we use `lib.optionals`.

Reply:
@AndersonTorres, yeah, that's what I did in https://github.com/NixOS/nixpkgs/pull/199978/commits/06ecc51368bb4181c313dd6fea1bc573b9b9a033.

Unfortunately it causes a 5000+package rebuild because `lib.optionals` evaluates to `[]` when the condition is `false` (rather than evaluating to `null`).  Ofborg puts the red ovals of doom on the PR, and then it gets ignored for a few years...

If you're willing to merge this PR in spite of the mass-rebuild just say so and I'll revert https://github.com/NixOS/nixpkgs/pull/199978/commits/8260aed123ea511cd10fb5a3385fe07cd12d349a.

Reply:
Given that it is just a fixup for a specific target, I till not touch such a wasp nest.